cloud-gcp/src/main/java/com/sequenceiq/cloudbreak/cloud/gcp/loadbalancer/GcpBackendServiceResourceBuilder.java

```diff
@@ -0,0 +1,139 @@
+package com.sequenceiq.cloudbreak.cloud.gcp.loadbalancer;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+import java.util.Map;
+import java.util.Optional;
+import java.util.Set;
+import java.util.stream.Collectors;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.stereotype.Service;
+
+import com.google.api.client.googleapis.json.GoogleJsonResponseException;
+import com.google.api.services.compute.Compute.RegionBackendServices.Delete;
+import com.google.api.services.compute.Compute.RegionBackendServices.Insert;
+import com.google.api.services.compute.model.Backend;
+import com.google.api.services.compute.model.BackendService;
+import com.google.api.services.compute.model.Operation;
+import com.sequenceiq.cloudbreak.cloud.context.AuthenticatedContext;
+import com.sequenceiq.cloudbreak.cloud.gcp.context.GcpContext;
+import com.sequenceiq.cloudbreak.cloud.model.CloudLoadBalancer;
+import com.sequenceiq.cloudbreak.cloud.model.CloudResource;
+import com.sequenceiq.cloudbreak.cloud.model.CloudStack;
+import com.sequenceiq.cloudbreak.cloud.model.Group;
+import com.sequenceiq.cloudbreak.cloud.model.TargetGroupPortPair;
+import com.sequenceiq.common.api.type.LoadBalancerType;
+import com.sequenceiq.common.api.type.ResourceType;
+
+/**
+ * Responsible for CRUD opertaions of a GCP Backend Service Resource
+ * A backend service could have multiple definitions, but currently the only one used is based on instance groups
+ * Set the health check mentod to be used for the related instance groups
+ */
+@Service
```

Comment:
Can we have logs with more details? 
I think we must have logging that will be handy at any time when error happens especially if it a producation issue or just would like to follow which resources have been created on the happy-path.

Reply:
@yg-gd can we have logs in create and delete methods too??

Reply:
I've added a general log statement for each resource builder as well as this specific builder